src/libraries/shims/ApiCompatBaseline.PreviousNetCoreApp.txt

```diff
@@ -1,170 +1,16 @@
-Compat issues with assembly netstandard:
```

Comment:
Hopefully we don't need this anymore.

Reply:
we have couple of different apis

Reply:
I meant the errors that you're removing 😄